src/sync.cpp

```diff
@@ -125,37 +132,52 @@ namespace librealsense
 
     void composite_matcher::dispatch(frame_holder f, const syncronization_environment& env)
     {
-        LOG_IF_ENABLE("DISPATCH " << _name << "--> " <<*f.frame, env);
-
         clean_inactive_streams(f);
         auto matcher = find_matcher(f);
 
+        //LOG_IF_ENABLE( "--> composite_matcher: " << _name, env );
+
         if (matcher)
         {
             update_last_arrived(f, matcher.get());
             matcher->dispatch(std::move(f), env);
         }
         else
         {
-            LOG_ERROR("didn't find any matcher for " << *f.frame << " will not be synchronized");
+            LOG_ERROR( "didn't find any matcher; frame will not be synchronized" );
```

Comment:
which frame?

Reply:
The frame we're syncing

Reply:
Has to be taken in context, in other words... I can add the frame...

Reply:
Added the frame